tests/sorting/test_heap_sort.py

```diff
@@ -0,0 +1,8 @@
+from algorithms.sorting.heap_sort import heap_sort
+
+
+class TestHeapSort:
```

Comment:
please add cases:
1) negative numbers
2) All zeros
3) Single item 
4) Empty array

And why do you use merge_sort instead heap_sort that you've implemented :)